src/app/tasks/create-task/create-task.component.ts

```diff
@@ -1,31 +1,35 @@
-import { Component, EventEmitter, Output } from '@angular/core';
-import { Task } from '../task.model';
-import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { Component, EventEmitter, Output, Input } from '@angular/core';
+import { Task, Option } from '../task.model';
+import { FormControl, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
 import { TaskService } from '../task.service';
-import { Validators } from '@angular/forms';
+import { KeyValuePipe } from '@angular/common';
 
 @Component({
   selector: 'app-create-task',
   standalone: true,
   imports: [
-    ReactiveFormsModule
+    ReactiveFormsModule,
+    KeyValuePipe
   ],
   templateUrl: './create-task.component.html',
   styleUrl: './create-task.component.scss'
 })
 export class CreateTaskComponent {
-  
+ 
   @Output() taskCreatedEvent = new EventEmitter<Task>();
 
+  @Input() taskTypes : Option[] = this.taskService.getTaskTypes();
```

Comment:
I noticed that this is an input, but it is not used as an Input, if you don't plan on using it as an Input you can remove the @Input